scripts/install.js

```diff
@@ -92,37 +92,47 @@ function download(url, dest, cb) {
  */
 
 function checkAndDownloadBinary() {
-  if (sass.hasBinary(sass.getBinaryPath())) {
+  if (process.env.SKIP_SASS_BINARY_DOWNLOAD_FOR_CI) {
+    console.log('Skipping downloading binaries on CI builds');
     return;
   }
+  var binaryPath = sass.getBinaryPath();
 
-  mkdir(path.dirname(sass.getBinaryPath()), function(err) {
+  if (sass.hasBinary(binaryPath)) {
+    return;
+  }
+
+  mkdir(path.dirname(binaryPath), function(err) {
```

Comment:
I'm not sure what `fs.stat` has to do with `mkdirp`? 

Having the vendor directory in the cache candidates list means that if we can't write to the npm cache path then we are writing to vendor. The fallback has already happened.